commcare_connect/templates/tailwind/components/worker_page/deliver_column.html

```diff
@@ -0,0 +1,42 @@
+{% load i18n %}
+{% load static %}
+{% load crispy_forms_tags %}
+
+<div class="relative underline"
+     x-data="{
+         isOpen: false,
+         positionMenu() {
```

Comment:
This appears to be used in multiple templates in this PR (and I imagine more outside this PR). Can it be pulled into some kind of shared js?

Reply:
Yeah, this is a good one to be pulled out, there are few more things like this. I am hoping to do these JS components in a post UI work PR.

Reply:
Is there a reason not to do this now?

Reply:
There are some open questions that needs to be answered to do these better, that I think could be part of latter refactor.
- Where should these common utils be placed in?
- Should they be included in the base-template?
- But since they aren't used in all pages, may be we will need a require_js setup. What are our options for this setup.

etc.